ArduPlane/Plane.h

```diff
@@ -1116,6 +1116,12 @@ class Plane : public AP_Vehicle {
         CROW_DISABLED,
     };
 
+    enum class ThrFailsafe {
+        Disabled    = 0,
```

Comment:
capitalisation is a bit wonky here.  OTOH, it does avoid ENABLED/DISABLED from config.h...